src/main/java/io/candydoc/ddd/extract_ddd_concepts/DDDConceptsExtractionService.java

```diff
@@ -64,13 +70,21 @@ private void trackAndApply(List<Event> occurredEvents) {
     occurredEvents.forEach(event -> event.accept(this));
   }
 
-  public void handle(ExtractValueObjects command) {
-    log.info("Extract value objects from {}", command.getPackageToScan());
-    trackAndApply(valueObjectExtractor.extract(command));
-  }
-
   public void handle(ExtractDDDConcepts command) {
     log.info("Extract ddd concepts from {}", command.getPackagesToScan());
+    for (String packageToScan : command.getPackagesToScan()) {
```

Comment:
You can also do it like this: command.getPackagesToScan().forEach(packageToScan -> ...)